### DISCUSSION TITLE
v0.4.1

### DISCUSSION THREAD

**Discussion by stefansundin** [stefansundin/privatkopiera#210] [Announcements]
- Fixade kompatibilitet med senaste versionerna av FFmpeg.
- Tillägget kan nu användas i Firefox för Android. Du kan nu privatkopiera på telefonen utan en dator! [Läs den här diskussionen för mer information!](https://github.com/stefansundin/privatkopiera/discussions/209)

**Full Changelog**: https://github.com/stefansundin/privatkopiera/compare/v0.4.0...v0.4.1

<hr /><em>This discussion was created from the release <a href='https://github.com/stefansundin/privatkopiera/releases/tag/v0.4.1'>v0.4.1</a>.</em>

**Comment by Danneboy74** [comment]
Hej! Har använt mig av privatkopiera tillägget förut och det har varit hur bra som helst.
Men efter att jag bytt dator och försökt installera ffmpeg så kan jag inte få det att fungera.
Nu får jag bara det här felmeddelandet när jag försöker använda tillägget.

Error opening output C:\Användare\danie\Das Boot - Avsnitt 7.mp4: No such file or directory
Error opening output file C:\Användare\danie\Das Boot - Avsnitt 7.mp4.
Error opening output files: No such file or directory

Vad sjutton har jag gjort för fel??

**Reply by stefansundin** [reply]
Hej. Det låter som att mappen du vill använda kanske inte finns?

Har du konfigurerat inställningen "Sökväg där filer sparas"? Min gissning är att mappen du har ställt in inte existerar.

Om du lämnar den inställningen tom så borde ffmpeg skriva filen till samma mapp som din kommandotolk befinner sig i (dvs det som står i början av prompt-raden där du klistrar in kommandot).

Hoppas det hjälper.

**Reply by Danneboy74** [reply]
Tack så hemskt mycket för hjälpen!
Ditt tips gjorde att det fungerade!
😊

**Comment by TwinTurbine** [comment]
Hej!

"Binär"-versionen av SVTpk (0.40) hittar strömmar med högre upplösning men omkodar allt vilket tar en evinnerlig tid. Tilläggsversionen för Firefox och Chrome (0.41) laddar ner blixtsnabbt men hittar inga strömmar med högre upplösning än 1080. Whiskey on the Rocks finns även i 2K vilket hittas av 0.40, men inte av tilläggen (0.41). Har provat i både Chrome och Firefox.

Vidare så laddas inte texten ner på samma gång, även om den senaste versionen av ffmpeg används. Dock går det fint att ladda ner texten separat så det är inte hela världen så sett. :-)

**Reply by stefansundin** [reply]
Nu med Privatkopiera v0.4.2 kan man ladda ner högre upplösning från SVT. Vet inte om programmet du nämner här har högre upplösningar dock.

**Comment by Arneby** [comment]
Idag söndag ser det ut som om SR byggt om sina URL:ar tyvärr. De verkar ha blivit kortare och renare vilket är snyggare men synd för oss som vill 'dokumentera'.

**Reply by Arneby** [reply]
ping!

**Reply by Arneby** [reply]
Stefan? Har du gett upp detta fantastiska tillägg?

**Reply by stefansundin** [reply]
SR är fixat i v0.4.2.
